io.github.parnassius.SysDVR_Qt.yml

```diff
@@ -0,0 +1,113 @@
+app-id: io.github.parnassius.SysDVR_Qt
```

Comment:
Shouldn't this be?

```suggestion
app-id: io.github.parnassius.SysDVR-Qt
```

Reply:
I'm not sure, but according to [this](https://docs.flatpak.org/en/latest/conventions.html#application-ids) I think application ids should use underscores instead of dashes. Am I missing something?

Reply:
Yes I'm sure. It doesn't apply to the last part (the app name)

Reply:
but then it has to match with the appstream file

Reply:
Oh, I see, I thought it applied to the whole thing. It should be fine now